doc/userguide/support-status.rst

```diff
@@ -0,0 +1,490 @@
+   +============+================+====+==========================+
+   |files-json  |                |    |Superceded by eve.fileinfo|
+   +------------+----------------+----+--------------------------+
+
+AppLayer Protocols
+~~~~~~~~~~~~~~~~~~
+
+Tier 1
+^^^^^^
+
+.. table::
+   :width: 100%
+
+   +------------+------------------------------------+----+-------------------+
+   |Protocol    |Maintainer                          |QA  |Notes              |
+   +============+====================================+====+===================+
+   |http        |Core team / Victor Julien           |    |includes libhtp    |
+   +------------+------------------------------------+----+-------------------+
+   |dns         |Core team / Jason Ish               |    |                   |
+   +------------+------------------------------------+----+-------------------+
+   |ssl/tls     |Mats Klepsland (Core team as backup)|    |                   |
+   +------------+------------------------------------+----+-------------------+
+   |smb/dcerpc  |Core team / Victor Julien           |    |                   |
+   +------------+------------------------------------+----+-------------------+
+   |smtp        |Core team                           |    |                   |
+   +------------+------------------------------------+----+-------------------+
+   |ssh         |Core team                           |    |                   |
+   +------------+------------------------------------+----+-------------------+
+   |dnp3        |Core team / Jason Ish               |    |                   |
+   +------------+------------------------------------+----+-------------------+
+
+Tier 2
+^^^^^^
+
+.. table::
+   :width: 100%
+
+   +------------+-------------------------+----+----------------------------------------------+
+   |Protocol    |Maintainer               |QA  |Notes                                         |
+   +============+=========================+====+==============================================+
+   |ftp         |Core team                |    |Currently only used for ‘ftpbounce’ keyword   |
+   +------------+-------------------------+----+----------------------------------------------+
+   |rust/nfs    |Core team / Victor Julien|    |Rust support currently experimental           |
+   +------------+-------------------------+----+----------------------------------------------+
+   |rust/dns    |Core team / Jason Ish    |    |Rust support currently experimental           |
+   +------------+-------------------------+----+----------------------------------------------+
+
+Community
+^^^^^^^^^
+
+.. table::
+   :width: 100%
+
+   +------------+-------------------+----+-------------------------+
+   |Protocol    |Maintainer         |QA  |Notes                    |
+   +============+===================+====+=========================+
+   |modbus      |David Diallo       |    |                         |
```

Comment:
was done in rust using sawp, need to think where it fits

Reply:
Left as community, no maintainer.